Cargo.toml

```diff
@@ -2,7 +2,9 @@
 members = [
     "examples/example-runner-cpu",
     "examples/example-runner",
+    "examples/wgpu-example-runner",
     "examples/example-shader",
+    "examples/wgpu-example-shader",
```

Comment:
Adding a comment here, one thing that I would change but I think could be done after merging, is to rename this shader to something like `hello-world-shader` or `simplest-shader` or something to that effect, since it shouldn't really be specific to wgpu.

Reply:
fully agree!

I was thinking that the current `example-shader` would become more, well simply, `shaders` with a set of non-trivial shaders like the current sky one and more in the same crate.

but also very good then to have a super simple separate crate that is easier to iterate on with fewer dependencies and features. esp. soon when glam gets in #149